query/sources/impl/src/main/java/org/codice/ddf/admin/sources/csw/CswFieldProvider.java

```diff
@@ -62,4 +56,24 @@ public CswFieldProvider(ConfiguratorSuite configuratorSuite) {
   public List<FunctionField> getMutationFunctions() {
     return ImmutableList.of(createCswConfiguration, updateCswConfiguration, deleteCswConfiguration);
   }
+
+  public void setGetCswConfigurations(GetCswConfigurations getCswConfigurations) {
```

Comment:
❓ I don't care much either way, but why are these properties instead of final fields initialized in the ctor?